coffee_core/src/coffee.rs

```diff
@@ -396,10 +396,20 @@ impl PluginManager for CoffeeManager {
     async fn list_remotes(&mut self) -> Result<CoffeeRemote, CoffeeError> {
         let mut remote_list = Vec::new();
         for repo in self.repos.values() {
+            let commit_id = match repo.git_head() {
+                Some(val) => val,
+                None => String::from(""),
+            };
+            let date = match repo.date_of_the_last_commit() {
+                Some(val) => val,
+                None => String::from(""),
+            };
```

Comment:
remove it and store inside the `CoffeeListRemote` the optional value

Reply:
this is outdated
solved in 3b55f3e